<launch>
  <arg name="system_params_path" default="$(find system)/config/system/pkgs/chrono.yaml"/>
  <arg name="vehicle_params_path" default="$(find system)/config/vehicle/hmmwv.yaml"/>
  <arg name="case_params_path" default="$(find system)/config/case/s1.yaml"/>
  <arg name="chrono_params_path" default="$(find system)/config/planner/chrono_testing/hmmwv_chrono_params.yaml"/>

  <rosparam command="load" file="$(arg system_params_path)"/>
  <rosparam command="load" file="$(arg vehicle_params_path)"/>
  <rosparam command="load" file="$(arg case_params_path)"/>
  <rosparam command="load" file="$(arg chrono_params_path)"/>
 
  <node name="vel_gen_chrono" pkg="traj_gen_chrono" type="vel_gen_chrono"/>
  <node name="velocity_controller" pkg="ros_chrono" type="velocity_controller" cwd="node"/>
</launch>
